charts/node-analyzer/values.yaml

```diff
@@ -273,7 +273,7 @@ nodeAnalyzer:
       deploy: false
       image:
         repository: sysdig/eveclient-api
-        tag: 1.0.0
+        tag: 1.1.0
```

Comment:
@ndsistsys there should be a line in the README that reference this version number, could you please update it too?